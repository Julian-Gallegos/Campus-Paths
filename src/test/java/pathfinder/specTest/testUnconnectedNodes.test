# Find a path with Dijkstra's algorithm when
# no path exists between the starting node and destination node

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

FindPath simpleGraph A B