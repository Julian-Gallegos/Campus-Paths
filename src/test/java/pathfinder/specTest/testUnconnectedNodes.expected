# Find a path with Dijkstra's algorithm when
# no path exists between the starting node and destination node

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

path from A to B:
no path found